bahmnicore-omod/src/main/java/org/bahmni/module/bahmnicore/web/v1_0/controller/TransmissionController.java

```diff
@@ -58,4 +57,4 @@ public Object sendEmail(@RequestBody BahmniMailContent bahmniMailContent, @PathV
         return response;
```

Comment:
this controller should be renamed to EmailPrescriptionController or so and exposing API /patient/<patient-uuid>/send-prescription-email or so